state/executor.go

```diff
@@ -189,7 +186,15 @@ func (e *BlockExecutor) ProcessProposal(
 		Txs:    block.Data.Txs.ToSliceOfBytes(),
 		ProposedLastCommit: abci.CommitInfo{
 			Round: 0,
-			Votes: []abci.VoteInfo{},
+			Votes: []abci.VoteInfo{
+				{
+					Validator: abci.Validator{
```

Comment:
why fill empty here? why not fetch it from `lastExtendedCommit` which has all the information?